src/nl/rubensten/texifyidea/run/LatexCompiler.java

```diff
@@ -48,15 +49,39 @@
             command.add("-interaction=nonstopmode");
             command.add("-synctex=1");
             command.add("-output-format=" + runConfig.getOutputFormat().name().toLowerCase());
-            command.add("-output-directory=" + moduleRoot.getPath() + "/out");
 
-            if (runConfig.hasAuxiliaryDirectories()) {
+            if (runConfig.hasOutputDirectories() && System.getProperty("os.name").contains("Windows")) {
+                command.add("-output-directory=" + moduleRoot.getPath() + "/out");
+            }
+
+            if (runConfig.hasAuxiliaryDirectories() && System.getProperty("os.name").contains("Windows")) {
                 command.add("-aux-directory=" + moduleRoot.getPath() + "/auxil");
             }
 
-            for (VirtualFile root : moduleRoots) {
-                command.add("-include-directory=" + root.getPath());
+            if (System.getProperty("os.name").contains("Windows")) {
+                for (VirtualFile root : moduleRoots) {
+                    command.add("-include-directory=" + root.getPath());
+                }
+            }
+
+        } else if (this == LUALATEX) {
```

Comment:
```suggestion
        }
        else if (this == LUALATEX) {
```

See [code conventions](https://github.com/Ruben-Sten/TeXiFy-IDEA/blob/master/CONTRIBUTING.md#code-conventions).

Reply:
Oh and remove the empty line above